README.md

```diff
@@ -37,6 +37,8 @@ App::new()
     .service(web::resource("/admin")
             .to(|| async { HttpResponse::Ok().finish() })
             .guard(PermissionGuard::new("ROLE_ADMIN".to_string())))
+    .service(web::resource("/admin") // in case you want to return 403
```

Comment:
```suggestion
    .service(web::resource("/admin") // in cases where you want to return HTTP 403 code
```

Reply:
Should I apply your suggestion somehow?))

Reply:
> Should I apply your suggestion somehow?))

You should commit this suggestion :)